src/components/App/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+
+const App = () => (
```

Comment:
I remember mentioning you guys should not create your components like this this morning

```suggestion
function App() {
  return (
```

Reply:
I'll fix it right now, thanks